### DISCUSSION TITLE
[Question]  Does this log messages?

### DISCUSSION THREAD

**Discussion by ghost** [victornpb/undiscord#340] [1. Questions & Answers] [answered]
> WARNING: Github Issues are publicly visible, do not post sensitive information here.
> You may want to use the "Screenshot mode" before taking screenshots or copying logs.

Describe what your question is, try to give as much detail as possible so I can help you.

Post screenshots if applicable.

Hello, I'm am an avid user of this program (for moderation and such), and I'd like a specification if any messages are logged to a different site, server, etc. After reading the code, I have yet to see anything very eye-catching or abnormal, and when you use Tampermonkey it will tell you "Connected websites: Discord.com" and nothing else. If you could specify this for me, that would be great! Thank you.

**Comment by victornpb** [comment, marked as answer]
NO. There are no external dependencies, and there are NO data collection or analytics of any kind.

Also, I encourage everyone to read: [What I'm doing to ensure this is safe for users](https://github.com/victornpb/undiscord/wiki/Security-Policy)

**Comment by victornpb** [comment]
That being said,

I only control the code found on **THIS** repository. (There are currently 169 forks, which I have no control.)

So as a general advice, I feel like I should give this disclaimer:

> **Running user scripts of any kind imposes a security risk, as they can steal personal information, impersonate and perform  actions on your behalf.**
> 
> **So always check the code before copying paste, or asking someone you trust.**
> 
>  I can't guarantee that someone will take this code, modify it for malicious intents and post somewhere else.
>
